### DISCUSSION TITLE
Removing `@tabler/icons-react` & `dayjs` peer dependencies

### DISCUSSION THREAD

**Discussion by enchorb** [KevinVandy/mantine-react-table#266] [Ideas]
Love the hard work that has gone into this project. 

Searching the code I don't see `dayjs` used unless I'm missing something (https://github.com/search?type=code&q=repo%3AKevinVandy%2Fmantine-react-table+from+%27dayjs%27)

Was thinking for the icons to allow user to pass in their own icons instead of installing another icon library (for those not already on `@tabler`

**Comment by alessandrojcm** [comment]
Hi there, `dayjs`  is a required Mantine [dependeny](https://mantine.dev/getting-started/) when using `@mantine/dates`, which this library does. And for `@tabler`, you can definetely passed your own icons; but if you do not MRT fallbacks to `@tabler`; otherwise users would have to always pass their own icons. If you aren not using tabler then your bundler should eliminate it anyway when tree shaking, @KevinVandy can correct me here if I am wrong.

**Reply by KevinVandy** [reply]
Yeah, the Mantine dates package is what depends on days.
And as of now, the tabler icons package is required. However, I've been thinking of adding a new useMantineReactTableLight hook for a long time where row models, locales, icons, and maybe a few other things would have to be manually specified. Maybe now is a good time to introduce that, even though I have much bigger plans with TanStack Table v9 in the future.
